### PR TITLE
Unexpected 'Project xxx has no explicit encoding set' warnings

### LINKED ISSUE #2416: Unexpected 'Project xxx has no explicit encoding set' warnings
In a Gradle project with subprojects, each subproject directory has a " Project 'xxx' has no explicit encoding set" warning. This appears to have started with version 1.5.0 of the extension.

##### Environment
- Operating System: macOS 12.3.1
- JDK version: OpenJDK 64-Bit Server VM Temurin-17.0.2+8 (build 17.0.2+8, mixed mode)
- Visual Studio Code version: Version: 1.67.0-insider Commit: a9288be67bab4e9c4dbe62207f01fafc2bdaadcb
- Java extension version: 1.5.0

##### Steps To Reproduce
1. Create basic project with Groovy DSL with `gradle init`
2. Add empty subproject directory in project
3. Add subproject directory to `include` in `settings.gradle`
4. Open project in vscode

[encoding-test.zip](https://github.com/redhat-developer/vscode-java/files/8497242/encoding-test.zip)

##### Current Result

Subproject directory has "Project 'xxx' has no explicit encoding set" warning.

##### Expected Result

No warnings.

### PR DESCRIPTION
**Pull request by snjeza**
Fixes #2416 
Requires https://github.com/eclipse/eclipse.jdt.ls/pull/2061

Signed-off-by: Snjezana Peco <snjezana.peco@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -199,7 +199,15 @@ The following settings are supported:
  * `java.quickfix.showAt` : Show quickfixes at the problem or line level.
  * `java.configuration.workspaceCacheLimit` : The number of days (if enabled) to keep unused workspace cache data. Beyond this limit, cached workspace data may be removed.
  * `java.import.generatesMetadataFilesAtProjectRoot` : Specify whether the project metadata files(.project, .classpath, .factorypath, .settings/) will be generated at the project root. Defaults to `false`.
- * `java.inlayHints.parameterNames.enabled`: Enable/disable inlay hints for parameter names. Supported values are: `none`(isable parameter name hints), `literals`(Enable parameter name hints only for literal arguments) and `all`(Enable parameter name hints for literal and non-literal arguments). Defaults to `literals`.
+ * `java.inlayHints.parameterNames.enabled`: Enable/disable inlay hints for parameter names. Supported values are: `none`(disable parameter name hints), `literals`(Enable parameter name hints only for literal arguments) and `all`(Enable parameter name hints for literal and non-literal arguments). Defaults to `literals`.
+
+New in 1.6.0
```
Comment (rgrunber) [inline README.md:204]: There's a recent addition here from https://github.com/redhat-developer/vscode-java/commit/c3b2b6ef4456155fdbe2404e2069376a839f62ca#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5 so I would just make sure to rebase on top of that. The `Defaults to ..` phrase can be right after the setting description.

Comment (snjeza) [reply README.md:205]: Fixed.

File: README.md [hunk 2]
```diff
@@ -200,10 +200,16 @@ The following settings are supported:
...
 
-New in 1.6.0:
+New in 1.6.0
 * `java.signatureHelp.description.enabled`: Enable/disable to show the description in signature help. Defaults to `false`.
+* `java.project.encoding`: Project encoding settings.
+  - `ignore`: Ignore project encoding settings.
+  - `warning`: Show warning if a project has no explicit encoding set.
+  - `setDefault`: Set the default workspace encoding settings.
+ 
+Defaults to `ignore`.
```
Comment (rgrunber) [inline README.md:212]: I would move this line (about the default) to 207, similar to what is done on 206. Another option is to simply state "(Default)"  at the end of the description for `ignore`.

Comment (snjeza) [reply README.md]: Fixed.

**Review by rgrunber — APPROVED** [review]

**Pull request merged**